telegram/ext/_jobqueue.py

```diff
@@ -51,6 +51,7 @@ class JobQueue:
     """
 
     __slots__ = ("_application", "scheduler", "_executor")
+    CRON_MAPPING = ["sun", "mon", "tue", "wed", "thu", "fri", "sat"]
```

Comment:
```suggestion
    _CRON_MAPPING = ("sun", "mon", "tue", "wed", "thu", "fri", "sat")
```
let's make it protected, as it's not intended to be used by users directly :) making it a tuple emphasizes that it's not entended to be changed.

Reply:
thanks @Bibo-Joshi  for the review, I learned a lot already in my first open source PR

Reply:
Does the pr need to be from my forks master?

Reply:
mh, no, you don't need a new branch or pr. see the "Resolve any merge conflicts that arise." step in the [contribution guide](https://docs.python-telegram-bot.org/en/v20.0a0/contributing.html#instructions-for-making-a-code-change)

Reply:
oh, update my branch's history, I pushed it now, luckily no conflicts at all